### PR TITLE
Clarifications of did:tdw features (abstract) and definitions

### PR DESCRIPTION
**Pull request by martipos**
references added, in the abstract and the definitions

SCID benefits summarized and description on portability added and separated from it, both in abstract

update of definitions for DID Log and Log Entries, and added definition for did:web

### CODE REVIEW

File: spec/abstract.md [hunk 1]
```diff
@@ -4,22 +4,24 @@ Trust DID Web (`did:tdw`) is an enhancement to the `did:web` DID method,
...
 - The same DID-to-HTTPS transformation as `did:web`.
 - Supports the same [High Assurance DID-to-DNS] mechanism.
 - The ability to resolve the full history of the DID using a verifiable chain of
   updates to the [[ref: DIDDoc]] from genesis to deactivation.
-- A [[def: self-certifying identifier]] (SCID) for the DID that is globally
-  unique, embedded in the DID, and derived from the initial [[ref: DIDDoc]]. The SCID
-  enables [[ref: DID portability]], such as moving the DID's web location (and
-  so changing the DID string itself) while retaining a connection to the
-  predecessor DID(s) and the DID's verifiable history.
+- A [[def: self-certifying identifier]] (SCID) for the DID. The SCID, globally unique and
```
Comment (swcurran) [inline spec/abstract.md:13]: This “def” should be a “ref” — my mistake.

File: spec/abstract.md [hunk 2]
```diff
@@ -4,22 +4,24 @@ Trust DID Web (`did:tdw`) is an enhancement to the `did:web` DID method,
...
+  embedded in the DID, is derived from the initial [[ref: DIDDoc]]. It ensures the integrity
```
Comment (swcurran) [inline spec/abstract.md:14]: Should be: “is derived from the initial [[ref: DID log entry]].”

File: spec/definitions.md [hunk 3]
```diff
@@ -30,14 +30,15 @@ by the controller of the DID.
 
 [[def: DID Log, DID Logs]]
 
-~ A log of JSON arrays each of which provides the information necessary to
-generate and validate a version of the [[ref: DIDDoc]] from the previous version.
+~ A DID Log is a list of [[ref: Entries]] one being added for each update of an entry item,
+including new versions of the [[ref: DIDDoc]] or changed information necessary to generate or validate the DID.
 
-[[def: DID [[ref: Log Entry]], [[ref: DID Log]] Entries]]
+[[def: Entry, Entries, DID [[ref: Log Entry]], [[ref: DID Log]] Entries]]
```
Comment (swcurran) [inline spec/definitions.md:36]: Arrgghh…those “refs” sneaking into the defs.  I’ll fix my “semi-automatic” tool to not do that in the future.  In the mean time, please remove the “ref” links in the “def” line.  Again — my mistake.

File: spec/definitions.md [hunk 4]
```diff
@@ -56,7 +57,12 @@ providers.
 
 [[def: DID:web]]
```
Comment (swcurran) [inline spec/definitions.md:58]: Should be lower case “DID” — my bad.

**Review by swcurran — CHANGES_REQUESTED** [review]
Looks good — a few tweaks, mostly from things that I had wrong in the first place. What about also adding your name to the editors list?

File: spec/abstract.md (hunk 2 above)
Comment (martipos) [reply spec/abstract.md:14]: uu, this was a nasty one.

File: spec/definitions.md (hunk 4 above)
Comment (martipos) [inline spec/definitions.md:58]:
```suggestion
[[def: did:web]]
```

**Comment by martipos** [comment]
messed up the DCO as the names used on github and locally seam not to be the same - thought they would be synced. my mistake. Will therefore close this PR and create a new one with the review comments addressed.

**Pull request closed without merging**